6_drivers_fetch.yml

```diff
@@ -49,10 +61,10 @@ targets:
     command: nldas_update_cells(nldas_cells, nldas_new_cells, '6_drivers/in/nldas_times.rds')
 
   nldas_new_cube_plan:
-    command: create_cube_task_plan(nldas_new_cube_files, ind_dir=I('6_drivers/log/nc'))
+    command: create_cube_task_plan(nldas_new_cube_files, ind_dir = cube_ind_dir)
 
   nldas_append_cube_plan:
-    command: create_cube_task_plan(nldas_append_cube_files, ind_dir=I('6_drivers/log/nc'))
+    command: create_cube_task_plan(nldas_append_cube_files, ind_dir = cube_ind_dir)
 
   "6_fetch_new_cubes_tasks.yml":
     command: create_cube_task_makefile(target_name, nldas_new_cube_plan, include = I("6_drivers_fetch.yml"), packages = I(c('dplyr','ncdf4','progress')), sources = '6_drivers/src/nccopy_utils.R')
```

Comment:
it seems that these two calls to `create_cube_task_makefile` have a lot of shared arguments - those could be pushed into `create_cube_task_makefile` so they'd only appear once. The consequence of making that change now should only be a rebuild of the two .yml files, at which point remake should realize that the .yml files weren't actually changed at all.

Reply:
good idea.